pkg/datapath/linux/node.go

```diff
@@ -509,9 +509,15 @@ func upsertIPsecLog(err error, spec string, loc, rem *net.IPNet, spi uint8) {
 	})
 	if err != nil {
 		scopedLog.WithError(err).Error("IPsec enable failed")
+		return fmt.Errorf("failed to enable ipsec with %s using local IP %s, rem %s, spi %d: %w",
```

Comment:
I think the error logging above (and in other places) is no longer needed, as it will be logged by the node manager.